[Bug]: [Nightly]Load partition respectively raises no exception now

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:5e74c60
- Deployment mode(standalone or cluster):standalone
- MQ type(rocksmq, pulsar or kafka):    rocksmq
- SDK version(e.g. pymilvus v2.0.0rc2):2.3.0.dev48
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```


### Current Behavior

Load partition respectively raises no exception now.
```python
@pytest.mark.tags(CaseLabel.L2)
def test_load_partiton_respectively(self):
    """
    target: test release the partition after load partition
    method: load partition1 and load another partition
    expected: raise exception
    """
    self._connect()
    collection_w = self.init_collection_wrap()
    partition_w1 = self.init_partition_wrap(collection_w)
    partition_w2 = self.init_partition_wrap(collection_w)
    partition_w1.insert(cf.gen_default_list_data())
    partition_w2.insert(cf.gen_default_list_data())
    collection_w.create_index(ct.default_float_vec_field_name, index_params=ct.default_flat_index)
    partition_w1.load()
    error = {ct.err_code: 5, ct.err_msg: f'load the partition after load collection is not supported'}
    partition_w2.load(check_task=CheckTasks.err_res,
                      check_items=error)
```
before: 
```
[2023-03-21 10:23:15 - DEBUG - ci_test]: (api_request)  : [Partition.load] args: [1, 20], kwargs: {} (api_request.py:56)
[2023-03-21 10:23:17 - DEBUG - ci_test]: (api_response) : None  (api_request.py:31)
[2023-03-21 10:23:17 - DEBUG - ci_test]: (api_request)  : [Partition.load] args: [1, 20], kwargs: {} (api_request.py:56)
[2023-03-21 10:23:17 - ERROR - pymilvus.decorators]: RPC error: [load_partitions], <MilvusException: (code=5, message=failed to load partitions, err=some partitions [440213095602798193] of collection 440213095602798182 has been loaded into QueryNode, please release partitions firstly[LoadParameterMismatched])>, <Time:{'RPC start': '2023-03-21 10:23:17.162664', 'RPC error': '2023-03-21 10:23:17.176005'}> (decorators.py:108)
```
now: 
raise no exception.

If it is a new setting, I will modify the test cases.

### Expected Behavior

raise exception

### Steps To Reproduce

_No response_

### Milvus Log

1. link: https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI/detail/master/317/pipeline/123/
2. log: [artifacts-milvus-standalone-nightly-317-pymilvus-e2e-logs.tar.gz](https://jenkins.milvus.io:18080/job/Milvus%20Nightly%20CI/job/master/317/artifact/artifacts-milvus-standalone-nightly-317-pymilvus-e2e-logs.tar.gz)
3. collection name: coll__Sh38oN99
4. failed time: [2023-03-20T18:04:40.691Z] [gw5] [ 20%] FAILED testcases/test_partition.py::TestPartitionParams::test_load_partiton_respectively

### Anything else?

_No response_

--- a/tests/python_client/testcases/test_insert.py
+++ b/tests/python_client/testcases/test_insert.py
@@ -1486,8 +1486,7 @@ class TestUpsertValid(TestcaseBase):
         res1 = collection_w.query(expr, [default_float_name], ["partition_1"])[0]
         assert [res1[i][default_float_name] for i in range(upsert_nb)] == float_values.to_list()
 
-    @pytest.mark.tags(CaseLabel.L2)
-    @pytest.mark.skip(reason="issue #22563")
+    @pytest.mark.tags(CaseLabel.L1)
     def test_upsert_same_pk_concurrently(self):
         """
         target: test upsert the same pk concurrently

--- a/tests/python_client/testcases/test_partition.py
+++ b/tests/python_client/testcases/test_partition.py
@@ -257,7 +257,7 @@ class TestPartitionParams(TestcaseBase):
         """
         target: test release the partition after load partition
         method: load partition1 and load another partition
-        expected: raise exception
+        expected: raise no exception
         """
         self._connect()
         collection_w = self.init_collection_wrap()
@@ -267,9 +267,7 @@ class TestPartitionParams(TestcaseBase):
         partition_w2.insert(cf.gen_default_list_data())
         collection_w.create_index(ct.default_float_vec_field_name, index_params=ct.default_flat_index)
         partition_w1.load()
-        error = {ct.err_code: 5, ct.err_msg: f'load the partition after load collection is not supported'}
-        partition_w2.load(check_task=CheckTasks.err_res,
-                          check_items=error)
+        partition_w2.load()
 
     @pytest.mark.tags(CaseLabel.L2)
     def test_load_partitions_after_release(self):

